Firestore/third_party/abseil-cpp/absl/base/config_test.cc

```diff
@@ -17,7 +17,7 @@
 #include <cstdint>
 
 #include "gtest/gtest.h"
-#include "absl/synchronization/internal/thread_pool.h"
+// #include "absl/synchronization/internal/thread_pool.h"
```

Comment:
Any reason to comment out rather than just deleting? (Also throughout the PR; I'll refrain from commenting on the others.)

Reply:
My intent had been to make conflicts easy to spot and understand if these sources are ever updated, but you're right, this is uglier than it needs to be.